go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -84,6 +84,21 @@ func newVtgateHandler(vtg *VTGate) *vtgateHandler {
 func (vh *vtgateHandler) NewConnection(c *mysql.Conn) {
 }
 
+func (vh *vtgateHandler) ComResetConnection(c *mysql.Conn) error {
+	ctx := context.Background()
+	session, _ := c.ClientData.(*vtgatepb.Session)
+	if session != nil {
+		if session.InTransaction {
+			defer atomic.AddInt32(&busyConnections, -1)
+		}
+		_, _, err := vh.vtg.Execute(ctx, session, "rollback", make(map[string]*querypb.BindVariable))
+		if err != nil {
```

Comment:
It's better to log the error and return success. In fact, you can have this function not return an error at all.